### PR TITLE
Update version of SWC

### PR DESCRIPTION
**Pull request by alshdavid**
Updated the version of SWC.

Not sure how to test this change, I ran:
```
cargo build
cargo test
```

I'm not sure how to consume this from [Deno CLI ](https://github.com/denoland/deno) locally for testing

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "deno_ast"
-version = "0.34.2"
+version = "0.34.3"
```
Comment (dsherret) [inline Cargo.toml:3]: Can you revert this patch version bump?

**Comment by dsherret** [comment]
Updating swc takes about a days worth of work because we need to test this in all dependent crates and make certain updates. Are there any specific fixes in swc that you need? We'll probalby do an swc upgrade again once https://github.com/swc-project/swc/pull/8736 lands

File: Cargo.toml (hunk 1 above)
Comment (alshdavid) [reply Cargo.toml]: Sure thing, reverted!

**Comment by alshdavid** [comment]
> Updating swc takes about a days worth of work because we need to test this in all dependent crates and make certain updates. Are there any specific fixes in swc that you need? We'll probalby do an swc upgrade again once [swc-project/swc#8736](https://github.com/swc-project/swc/pull/8736) lands (we do an swc upgrade about once a month)

Hey, thanks for reviewing my change. My motivation for bumping swc is that my main project already uses a newer version of swc and consuming `deno_ast` causes a versioning conflict that Cargo cannot resolve. 

That said I have managed to isolate Deno (and its dependencies) from my main application so this is no longer a concern and there is no urgency to merge this PR from my end.

**Pull request closed without merging**